core/scss/components/button-group/module.scss

```diff
@@ -1,15 +1,38 @@
-@import '../../tools/functions/core';
+@import '../../tools/mixins/core';
 
-@import '../../colors/settings';
-@import '../../settings/core';
-@import '../../settings/media';
-@import '../../typography/settings';
-@import '../button/settings';
-@import '../../tools/functions/core';
+// Settings
+@import '../../settings/settings';
 
+// Tools
 @import '../../tools/mixins/border-radius';
+@import '../../tools/mixins/flex';
+@import '../../tools/mixins/focus.scss';
+
+// Generic
+@import '../../generic/global';
```

Comment:
Do we need all of the normalize and global CSS for button group? Can we instead just add the individual CSS that is needed?